examples/file-feed/complete/src/browser-app/index.css

```diff
@@ -0,0 +1,24 @@
+@import url('https://fonts.googleapis.com/css?family=Anonymous+Pro|Source+Code+Pro');
```

Comment:
I'd like to understand your reasoning here, can you elaborate? Why should we not fetch the fonts externally?

I'll add a fallback to the correct system fonts, though.